benchmarking/README.md

```diff
@@ -0,0 +1,6 @@
+Considerations for benchmarking:
+1. demonstrate that particle size only matters for the initial diffraction volume calculation
+2. assess how SPI, SPI aggregate, and FXS/holo experiments scale (no. images) with:
+a. parallelization / number of GPUs
+b. detector size 
+c. certain types of noise, e.g. beam miscentering?
```

Comment:
other things that come to mind that could affect timing is the mesh size used for the hydration layer